sys/net/include/nomac.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    nomac Link layer with no medium access
+ * @ingroup     net
+ * @brief       Link layer protocol to speak with any transceiver driver
+ *              directly without any medium access.
+ * @{
+ *
+ * @file
+ * @brief       Link layer protocol to speak with any transceiver driver
+ *              directly without any medium access.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#ifndef __NOMAC_H_
+#define __NOMAC_H_
+
+#include <stdlib.h>
+
+#include "netdev/base.h"
+#include "netapi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef NOMAC_REGISTRY_SIZE
+/**
+ * @brief   The size of NOMAC's registry of receiving threads.
+ */
+#define NOMAC_REGISTRY_SIZE (1)
```

Comment:
I would not make this parameter static, but allocate it on the nomac thread's stack and give it as parameter during initialization...